internal/bminventory/inventory.go

```diff
@@ -2535,6 +2535,7 @@ func (b *bareMetalInventory) updateNetworkTables(db *gorm.DB, cluster *common.Cl
 func (b *bareMetalInventory) updatePlatformParams(params installer.V2UpdateClusterParams, updates map[string]interface{}, usages map[string]models.Usage) error {
 	if params.ClusterUpdateParams.Platform != nil && common.PlatformTypeValue(params.ClusterUpdateParams.Platform.Type) != "" {
 		updates["platform_type"] = params.ClusterUpdateParams.Platform.Type
+		updates["platform_is_external"] = swag.BoolValue(params.ClusterUpdateParams.Platform.IsExternal)
```

Comment:
Should call common.isPlatformExternal() - user is not setting it

Reply:
I update the flag in `GetActualUpdateClusterPlatformParams` / `GetActualCreateClusterPlatformParams`, those methods create  a valid `Platform` struct out of `ClusterUpdateParams`/`ClusterCreateParams`.

Reply:
Right, thanks